src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+//TODO: Decide about i18n solution or simply a JSON file where we can load translations.
+
+const TRANSLATIONS = {
+  SHIP_TO_ME: 'Ship to Me',
+  ARRIVES: 'Arrives'
+};
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+`;
+Container.displayName = 'Container';
+
+//TODO:PLACEHOLDER FOR ICON.
+const Icon = styled.div`
```

Comment:
Don't make 'placeholders'. It is better to not show something rather than showing something half-baked.